README.md

```diff
@@ -299,6 +315,38 @@ The value for `SPEED` must be a valid positive integer.
 
 ---
 
+### test
+
+Verifies if the MediaRemote Adapter is functioning correctly.
+
+This can be integrated into your app to help confirm that our adapter is still effective and select fallback options if necessary since future macOS updates may break it again.
+
+**Usage**
+
+```
+/usr/bin/perl /path/to/mediaremote-adapter.pl /path/to/MediaRemoteAdapter.framework /path/to/NowPlayingTestClient test
+```
+
+Note that the `test` command requires the absolute path to the `NowPlayingTestClient` helper executable as the second argument (before the command name).
+
+**Output**
+
+The command outputs a single character to stdout:
+- `0` - MediaRemote Adapter is working correctly
+- `1` - MediaRemote Adapter is not working correctly
```

Comment:
I think we should just simplify this to exit codes, no stdout output. 0 for "success" and any other exit code means the adapter is not functioning on this particular system.